README.md

```diff
@@ -78,4 +78,25 @@ Consumer.PlainSource(consumerSettings, subscription)
     {
         Console.WriteLine($"Consumer: {result.Topic}/{result.Partition} {result.Offset}: {result.Value}");
     }, materializer);
-```
\ No newline at end of file
+```
+
+### Committable Consumer
+The `Consumer.CommittableSource` makes it possible to commit offset positions to Kafka.
+
+Compared to auto-commit this gives exact control of when a message is considered consumed.
+
+If you need to store offsets in anything other than Kafka, `PlainSource` should be used instead of this API.
+
+This is useful when “at-least once delivery” is desired, as each message will likely be delivered one time but in failure cases could be duplicated.
+
+```C#
+Consumer.CommitableSource(consumerSettings, Subscriptions.Topics("topic1"))
```

Comment:
I guess it should be `Subscriptions.Topic` not `Topic` (since we're picking a single topic, plural form doesn't sound right).

Reply:
It could accept one or many topics `Topics(params string[] topics)`